src/PublicKey/Rsa/PublicKey.php

```diff
@@ -131,27 +122,16 @@ public function decrypt($data, $padding = OPENSSL_PKCS1_PADDING)
         return $decrypted;
     }
 
-    /**
-     * Get certificate string
-     *
-     * @return string
-     */
-    public function getCertificate()
-    {
-        return $this->certificateString;
-    }
-
```

Comment:
This removal should probably be documented

Reply:
is there something like a changelog or will it be manually filled with a release of a new tag?

Reply:
Just searched - There's no mention of it in any docs or any usage anywhere else, not even in tests.

As it's not mentioned in the docs, perhaps all that is required is a mention in the release notes. I don't think this is a big deal👍

Reply:
I tend to add stuff like this to the milestone manually using keep-a-changelog format.
did that a bunch of times for laminas-cache and related adapters.